data/data/aws/variables-aws.tf

```diff
@@ -199,7 +199,7 @@ variable "tectonic_aws_region" {
   description = "The target AWS region for the cluster."
 }
 
-variable "tectonic_aws_installer_role" {
+variable ":ectonic_aws_installer_role" {
```

Comment:
spelling mistake?

Reply:
mind and fingers not cooperating :). thanks.